### ISSUE TITLE
How to create sibling directory or file if g.nvim_tree_group_empty is enabled?

### ISSUE THREAD

**Issue by dzintars** [kyazdani42/nvim-tree.lua#577]
`vim.g.nvim_tree_group_empty = 1` is enabled.
Cursor position on `..../dir-a/dir-b/dir-c`
Hit `a` to add new `dir-b` sibling and prompt shows:
```sh
Create file ..../dir-a/dir-b/dir-c/
```
How do i go one directory up and type:
```sh
Create file ..../dir-a/dir-b1
```
?
Personally i can't delete `/dir-c/` because backspacing does not work for me.
What i am missing?

**Comment by kyazdani42** [comment]
this is something i struggle with too, i usually use the command line to create directories in this case which is not ideal. I might just implement something at some point

**Comment by kyazdani42** [comment]
closed in `1.2.4` (425f5ee) you can now add files anywhere on the system, you can remove the folder name to create a file between two grouped folders

**Comment by dzintars** [comment]
This is super cool and seems working flawlesly!!!
Thank You for your efforts!!
Actually this was the last issue which blocked my personal switch from NERDTree. :)
